backend/src/main/java/com/hyooun/wilson/review/domain/Change.java

```diff
@@ -0,0 +1,19 @@
+package com.hyooun.wilson.review.domain;
+
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+
+@Getter
+@AllArgsConstructor
+public class Change {
+
+	private final String filePath;
+	private final int lineNumber;
+	private final String oldCode;
+	private final String newCode;
+	private final ChangeType type;
```

Comment:
Change 클래스에 ChangeType enum 추가 필요.

```suggestion
public enum ChangeType { ADD, DELETE, MODIFY }
```